### ISSUE TITLE
Next JS abort(LinkError: import object field '$' is not a Function). Build with -s ASSERTIONS=1 for more info.

### ISSUE THREAD

**Issue by saad4software** [IFCjs/web-ifc-viewer#174]
refactoring the simple react sample to fit Next JS, I ended up with:
```js
        const container = document.getElementById('viewer-container');
        const IfcViewerAPI = (await import("web-ifc-viewer")).IfcViewerAPI;
        const IfcLoader = (await import("web-ifc-three/IFCLoader")).IFCLoader;

        const viewer1 = new IfcViewerAPI({container});
        viewer1.axes.setAxes();
        viewer1.grid.setGrid()
        await viewer1.IFC.setWasmPath('../../../');
        await viewer1.IFC.loadIfcUrl("haus.ifc")
```
this code showed the grid but loading a model produced this issue 

![Screenshot from 2023-01-11 21-05-21](https://user-images.githubusercontent.com/85710350/211885543-2ccf20a6-3de1-400e-8d56-9d687c8bfa7d.png)

I have even imported the view without ssr in the main page, but it didn't solve the issue
```js
    const Ifc = dynamic(() => import('components/ifc'), {
        ssr: false
    })

```
I have faced similar issue in Nuxt and got around it, but I failed to find my path around this one
please help

**Comment by agviegas** [comment]
Hi, are you serving the `wasm` files correctly in your applications?

**Comment by agviegas** [comment]
Is [this](https://github.com/IFCjs/web-ifc-viewer/issues/173#issuecomment-1383566077) the same question?

**Comment by saad4software** [comment]
no, next js is a react framework while nuxt js is a vue framework. I have tried next as after I failed to run it in nuxt, I'll share a basic project (similar to the one with nuxt) if it helps

**Comment by saad4software** [comment]
actually it worked with the new version of wasm files, thanks again
I'll close this issue
